### ISSUE TITLE
2.3.0: missing config/getlib.sh in dist tar ball

### ISSUE THREAD

**Issue by kloczek** [libgd/libgd#610]
By this gd_LT_CURRENT, gd_LT_REVISION and gd_LT_AGE are empoty and linking fails with:
```
[tkloczko@barrel libgd-2.3.0]$ make
Making all in src
make[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libgd-2.3.0/src'
make  all-am
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/libgd-2.3.0/src'
/bin/sh ../libtool  --tag=CC   --mode=link gcc  -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -flto=auto -flto-partition=none -DDEFAULT_FONTPATH='"/usr/share/fonts/bitstream-vera:/usr/share/fonts/dejavu:/usr/share/fonts/default/Type1:/usr/share/X11/fonts/Type1:/usr/share/fonts/liberation"' -fvisibility=hidden -version-info :: -no-undefined -Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -flto=auto -flto-partition=none -fuse-linker-plugin -o libgd.la -rpath /usr/lib64 gd.lo gd_bmp.lo gd_color.lo gd_color_map.lo gd_color_match.lo gd_crop.lo gd_filename.lo gd_filter.lo gd_gd.lo gd_gd2.lo gd_gif_in.lo gd_gif_out.lo gd_interpolation.lo gd_io.lo gd_io_dp.lo gd_io_file.lo gd_io_ss.lo gd_jpeg.lo gd_matrix.lo gd_nnquant.lo gd_png.lo gd_rotate.lo gd_security.lo gd_ss.lo gd_tga.lo gd_tiff.lo gd_topal.lo gd_transform.lo gd_version.lo gd_wbmp.lo gd_webp.lo gd_xbm.lo gdcache.lo gdfontg.lo gdfontl.lo gdfontmb.lo gdfonts.lo gdfontt.lo gdft.lo gdfx.lo gdhelpers.lo gdkanji.lo gdtables.lo gdxpm.lo wbmp.lo  -lm   -lz   -lpng16 -lz   -lfreetype   -lfontconfig -lfreetype   -ljpeg    -limagequant -fopenmp  -lXpm -lX11   -ltiff    -lwebp
libtool:   error: CURRENT '' must be a nonnegative integer
libtool:   error: '::' is not valid version information
make[2]: *** [Makefile:777: libgd.la] Error 1
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libgd-2.3.0/src'
make[1]: *** [Makefile:634: all] Error 2
make[1]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/libgd-2.3.0/src'
```

**Comment by kloczek** [comment]
After switching to autogenerated tar from git tar https://github.com/libgd/libgd/archive/gd-2.3.0.tar.gz everything is fine.

Could you please confider change git tagging convention to use in git tag only version?
With your curretn convention tagged tarball is with `libgd-gd-<version>` base directory because git automatically adds to that directory git repo name prefix.

**Comment by willson-chen** [comment]
@kloczek hi. Thanks for your feedback.

Indeed `config/getlib.sh` doesn't exist in dist tar ball. But I can't not reproduce the error you encountered. Could you please specify the procedure to reproduce it?

> Could you please confider change git tagging convention to use in git tag only version?
With your curretn convention tagged tarball is with libgd-gd-<version>/ base directory because git automatically adds to that directory git repo name prefix.

Yeah, `libgd-gd-<version>` seems to a little strange. But considering the convention since our first tag, I prefer keeping the tag naming rule.

**Comment by kloczek** [comment]
Just try to regenerate ac/am/lt files using `autorevonf -fiv` before execute configure.

**Comment by willson-chen** [comment]
This issue is fixed in commit [635dd9a](https://github.com/libgd/libgd/commit/635dd9a3065ed88e1741e6b963044b80e913f96a).
